File: data/src/main/java/io/forus/me/android/data/entity/records/response/RecordType.java
package io.forus.me.android.data.entity.records.response;

import com.google.gson.annotations.SerializedName;

public class RecordType {

    @SerializedName("key")
    private String key;

    @SerializedName("type")
    private String type;

    @SerializedName("name")
    private String name;

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: domain/src/main/java/io/forus/me/android/domain/models/records/RecordType.kt
package io.forus.me.android.domain.models.records

import io.forus.me.android.domain.models.currency.Currency
import java.util.*

class RecordType {



    var id: String = ""

    var name: String = ""


    var logo: String = ""


    constructor(id: String, name: String, logo: String) {
        this.id = id
        this.name = name
        this.logo = logo
    }
}
